src/storekit.cs

```diff
@@ -132,34 +140,39 @@ interface SKMutablePayment {
 		[Availability (Deprecated = Platform.iOS_5_0, Message = "Use 'PaymentWithProduct (SKProduct)' after fetching the list of available products from 'SKProductRequest' instead.")]
 		SKMutablePayment PaymentWithProduct (string identifier);
 
+		[Watch (6, 2)]
```

Comment:
move to type level

it's important because now that StoreKit is included (in `Xamarin.WatchOS.dll`) it means that the lack of attribute correspond to the earliest supported version (i.e. watchOS 2.0).